pkg/plugins/policies/meshtrafficpermission/plugin/v1alpha1/testdata/apply-egress.golden.yaml

```diff
@@ -13,9 +13,9 @@ resources:
         - external-service-1{mesh=mesh-1}
         transportProtocol: tls
       filters:
-      - name: envoy.filters.network.rbac
+      - name: envoy.filters.http.rbac
```

Comment:
have you checked that it actually works? I thought that HTTP filters have to be placed in httpFilters inside of HTTP connection manager

Reply:
Good catch.

Reply:
addressed